docs/pages/application-access/guides/dynamodb.mdx

```diff
@@ -135,24 +135,41 @@ Replace `https://console.aws.amazon.com` with
 </Admonition>
 
 ### Install and start Teleport
+
 Install Teleport on the host where you will run the Teleport Application
 Service. See our [Installation](../../installation.mdx) page for options
 besides Linux servers.
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Now start the Teleport Application Service using the output from the previous
-step:
+Edit the Teleport configuration file (`/etc/teleport.yaml`) to include the
+following information, adjusting the value of `proxy_server` to specify the host
+and port of your Teleport Proxy Service:
 
-```code
-$ teleport app start \
-   --token=(=presets.tokens.first=) \
-   --ca-pin=(=presets.ca_pin=) \
-   --auth-server=https://teleport.example.com:443 \
-   --name=aws-dynamodb \
-   --uri=https://console.aws.amazon.com/dynamodbv2/home
+```yaml
+version: v3
+teleport:
+  join_params:
+    token_name: "/tmp/token"
+    method: token
+  proxy_server: "teleport.example.com:443"
+auth_service:
+  enabled: off
+proxy_service:
+  enabled: off
+ssh_service:
+  enabled: off
+app_service:
+  enabled: true
+  apps:
+  - name: aws-dynamodb
+    uri: https://console.aws.amazon.com/dynamodbv2/home
```

Comment:
IIRC this is going to become a first-party database in Teleport 12 @r0mant? Should we leave this in for now to support legacy use cases?

Reply:
Since we still support the Application Service approach, I thought I would refactor the existing doc to use systemd-based instructions for starting Teleport with AWS credentials. If this is now in "unsupported hack" territory, I suppose we can delete this guide, though perhaps as another PR.

Reply:
I do think we should raise a separate PR to remove this recommendation and instead link to the new DynamoDB guide for database access (https://goteleport.com/docs/database-access/guides/aws-dynamodb/) - not a top priority though.

Reply:
https://github.com/gravitational/teleport/issues/21774